cardano-api/src/Cardano/Api/Orphans.hs

```diff
@@ -470,3 +472,15 @@ instance (Ledger.Era era, Show (Ledger.Value era), ToJSON (Ledger.Value era))
            ]
 
 deriving instance Show Alonzo.AlonzoGenesis
+
+-- Rationals and JSON are an awkward mix. We cannot convert rationals
+-- like @1/3@ to JSON numbers. But _most_ of the numbers we want to use
+-- in practice have simple decimal representations. Our solution here is
+-- to use simple decimal representations where we can and representation
+-- in a @{"numerator": 1, "denominator": 3}@ style otherwise.
+--
+toRationalJSON :: Rational -> Aeson.Value
+toRationalJSON r =
+  case Scientific.fromRationalRepetend (Just 5) r of
```

Comment:
Same thing, but a bit more specific:
```suggestion
  case Scientific.fromRationalRepetendLimited 5 r of
```

I'd also suggest increasing the limit from 5 to say 20. Bounded rationals are represented by a Word64, which means that they support up to 19 places after the decimal point.

Reply:
I've switched to `20` as suggested.